mexican-restaurant-page.html

```diff
@@ -0,0 +1,87 @@
+        }
+        .left {
+            width: 50%;
+        }
+        .middle {
+            width: 25%;
+        }
+        .right {
+            width: 25%;                
+        }
+        ul {
+            list-style-type: none;
+        }            
+        h4 {
+            position: relative;
+            display: inline;
+            top: 15px;
+        }
+        .social a {                
+            display: inline;               
+        }            
+    </style>
+</head>
+<body>
+    <!-- there is a arrow that allows you to scroll the first column -->
+    <div class="container">
+        <div class="row">
+            <div class="column">
+                <ul>
+                   <li><img src="#" alt="picture of tacos and a beverage"></li>
+                    <!-- img over laps image from above -->
+                    <!-- img needs to be under the first image -->
+                    <li><img src="#" alt="picture of open tacos and salsa to the side"></li>
+                </ul>                   
+            </div>
+        </div>
+    </div>
+    <div class="container">
+        <div class="row">
+            <div class="column">
+                <ul>
+                    <li><a href="#Burrito">Burrito, Bowels & Quesadillas</a></li>
+                    <li><a href="#Tacos">Tacos</a></li>
+                    <!-- salate not sure if its a typo but that is what is on site -->
+                    <li><a href="#Salate">Salate</a><li>
+                    <li><a href="#Dessert">Dessert</a></li>
+                    <li><img src="#" alt="logo"></li>
+                    <!-- styling that when click on phone number gives you pop up option to call -->
+                    <li><a href="url">089 27373381</p><li>
+                    <!-- before e is a special character -->
+                    <li><a href="url">ElisabethstraBe 12, 80796</a></li>
+                    <li>Munchen</li>
+                    <li>Di-Do 18:00-00:00</li>
+                    <li>Fr-Sa: 18:00-02:00</li>
+                </ul>
+            </div>
+        </div>
+    </div>
+    <div class="container">
+        <div class="row">
+            <div class="column">
+                <div class="social">
+                    <h4>News</h4>
+                <!-- CSS to make social links line up with News -->
+                    <a href="url">Facebook</a>
+                    <a href="url">Instagram</a>                       
+                </div>
+            <br>
+    <div class="container">
+        <div class="row">
+            <div class="column">
+                <img src="#" alt="picture of burrito on a plate">
+                <!-- between Happy and Day is an png of burrito -->
+                <p>Happy day</p>
+                <img src="#" alt="picture of three Tacos">
+                <p>Taco Mania Mix & Match @iss.es#newonthemenu #MixIT #mixyourownchoice</p>
+            </div>
+        </div>
+    </div>            
+</body>
+
```

Comment:
Missing closing `<html>` tag

Reply:
I don’t have an opening HTML tag. My understanding was if you use <!DOCTYPE html>, you do not have to have <HTML>. As <!DOCTYPE html> is a declaration tag it does not have to be closed.

Reply:
Line 1 is: `<!DOCTYPE html>`, which needs to be closed with a closing tag.  
Please see below:
https://www.w3schools.com/tags/tag_doctype.asp

Normally self-closing tags like `<p/>` don't require a closing tag.
However, in this case, `<!DOCTYPE html>` is not self-closing.